src/sentry/sentry_metrics/configuration.py

```diff
@@ -9,6 +9,15 @@ class UseCaseKey(Enum):
     RELEASE_HEALTH = "release-health"
     PERFORMANCE = "performance"
 
+    @staticmethod
+    def from_str(use_case: str) -> "UseCaseKey":
+        if use_case == "performance":
+            return UseCaseKey.PERFORMANCE
+        elif use_case in ("release-health", "releaseHealth"):
+            return UseCaseKey.RELEASE_HEALTH
+        else:
+            return UseCaseKey.RELEASE_HEALTH
```

Comment:
Do we want to default in the `else` here? or should we raise an exception if it is not one of those use cases?

Reply:
yeah originally i had exception, but since it's optional ... i'm not 100% sure what we want here. Do you want me to raise `NotImplementedError` or something more specific?

Reply:
@ahmedetefy i decided to add raise exception: pls check it here 104af64353